experimental/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ All notable changes to experimental packages in this project will be documented
 
 ### :bug: (Bug Fix)
 
+* fix(prometheus-exporter): add possibility respond to errors returned by `server.listen()` [#3552](https://github.com/open-telemetry/opentelemetry-js/pull/3402) @pichlermarc
```

Comment:
grammar nitpick: should be "add possibility to respond to errors"

Reply:
Thanks for the feedback :slightly_smiling_face: 
I adapted both the entry and the title ([f4e226d](https://github.com/open-telemetry/opentelemetry-js/pull/3552/commits/f4e226ddc7664a34349bc763747b73b84634f71e)) :slightly_smiling_face: